README.md

```diff
@@ -100,18 +99,15 @@ module.exports = createConfig([
     }),
     sourceMaps()
   ]),
-  env('production', [
-    uglify(),
-    addPlugins([
-      new webpack.LoaderOptionsPlugin({ minimize: true })
-    ])
-  ])
+  env('production', [uglify(), addPlugins([new webpack.LoaderOptionsPlugin({ minimize: true })])])
```

Comment:
It makes JS code less readable in some cases 😞

Reply:
We can wrap such cases with `prettier-ignore`: https://prettier.io/docs/en/ignore.html#range-ignore